compiler/one-cmds/one-prepare-venv

```diff
@@ -34,8 +34,8 @@ fi
 # - https://github.com/onnx/onnx-tensorflow/blob/master/Versioning.md
 
 VER_TENSORFLOW=2.12.1
-VER_ONNX=1.14.0
-VER_ONNXRUNTIME=1.15.0
+VER_ONNX=1.16.0
+VER_ONNXRUNTIME=1.17.3
```

Comment:
onnxruntime 1.17.3 does not support onnx 1.16.0 yet.

From cmake/deps.txt in 1.17.3 source code.

```
onnx;https://github.com/onnx/onnx/archive/refs/tags/v1.15.0.zip;54c3f960a0541c5d8d3e60c2933e11f5d3688a11
```